### DISCUSSION TITLE
Right-to-Left flow

### DISCUSSION THREAD

**Discussion by GrimPixel** [IndiePython/nodezator#70] [Feature Request]
There are writing systems in right-to-left direction. When they are supported later, the interface will need that option as well.
There are some vertical writing systems, however, given that the program is usually run on a desktop, where the screen is wide and not tall, up-to-down flow doesn't fit the screen and can be ignored.

**Comment by KennedyRichard** [comment]
Yes, you are right, I agree. I'll take this into account when enabling more writing systems in Nodezator in the future. Thank you for the request.
